scripts/list.formulae

```diff
@@ -2,14 +2,19 @@
 
 set -xe
 
+# OS we're testing formulae on
+os="${1:-DUMMY_OS_VALUE}"
```

Comment:
@asmaahassan90 he is sending the OS variable and an argument to the file up there.

@antoniokaust I think you are trying to cut the OS version part from the variable, right ?!

Reply:
I just want it to work even when you test it on the command line without passing any arguments so it should list all the formulae. So I just set os to `DUMMY_OS_VALUE` when you don't specify anything.

Reply:
so, it's defined with a dummy value, and if I specified certain OS it gets overridden, else it continues with the dummy value, correct?

Reply:
Yes. The idea is not to leave it empty otherwise the second grep in the pipe always succeeds. This means you cannot use `DUMMY_OS_VALUE` as an OS name.